app/src/main/java/org/mozilla/focus/browser/WebViewCache.kt

```diff
@@ -42,4 +45,10 @@ class WebViewCache {
         cachedView?.removeFromParentIfAble()
         return cachedView ?: createAndCacheEngineView()
     }
+
+    @OnLifecycleEvent(Lifecycle.Event.ON_DESTROY)
+    private fun clear() {
```

Comment:
nit: call `onDestroy`; calling it clear will discourage other people from using `onDestroy` as `onDestroy`, thinking it might be just for `clear`